Wikisource.js

```diff
@@ -80,11 +180,9 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"abstractNote": "\"Amazing Grace\" (c. 1772) is one of the most well-known Christian hymns. The words thereof are part of the Olney Hymns that the author had worked on, with William Cowper and other hymnodists.\nThe text was first set to the tune NEW BRITAIN in William Walker's Southern Harmony, 1835.",
```

Comment:
1) I think it might rather be an exception that there is something like an abstract in wikisource. I am not even sure if the note here is really an abstract or maybe more like a description of this famous text.

2) Previously, this was extracted from the node `//div[contains(@class, "header_notes")]` which will also result in "Listen to this text (help | file info or download)" for https://en.wikisource.org/wiki/Alice's_Adventures_in_Wonderland_%281866%29/Chapter_2